projects/packages/forms/src/blocks/field-textarea/edit.js

```diff
@@ -25,6 +25,7 @@ export default function TextareaFieldEdit( props ) {
 
 	const defaultLabel = __( 'Message', 'jetpack-forms' );
 
+	// TODO: this label attribute is no longer provided, should we remove it?
```

Comment:
I believe this was to match prior behaviour that was previously handled via a `useEffect`. A dedicated issue can be created to investigate this potential clean up. I don't think it needs to be a blocker for release of the inner blocks refactor efforts.

Reply:
Here's a link to the specific discussion/PR - https://github.com/Automattic/jetpack/pull/43036/files#r2039242208